server/publications/collections/shops.js

```diff
@@ -12,6 +12,15 @@ Meteor.publish("PrimaryShop", function () {
   });
 });
 
+Meteor.publish("Shop", function () {
```

Comment:
Curious to understand what the change here is intended to do and why the old publication was left alone even though it's subscription was removed.

Reply:
oops! just an oversight